### DISCUSSION TITLE
Is wiki.js affected?

### DISCUSSION THREAD

**Discussion by var-log-skynet** [requarks/wiki#3781] [General]
I'm no developer so this might sound lame, but is wiki.js affected by this?

https://sick.codes/universal-netmask-npm-package-used-by-270000-projects-vulnerable-to-octal-input-data-server-side-request-forgery-remote-file-inclusion-local-file-inclusion-and-more-cve-2021-28918/

**Comment by PaulD987** [comment]
netmask isn't listed in package.json, and not in node_modules from an install, so I would say no it isn't affected.

**Reply by var-log-skynet** [reply]
Thank you!
